istio/src/main/java/com/alibaba/nacos/istio/misc/IstioConfig.java

```diff
@@ -32,8 +32,16 @@ public class IstioConfig {
     private boolean serverEnabled = false;
     @Value("${nacos.istio.mcp.server.port:18848}")
     private int serverPort = 18848;
-    @Value("${nacos.istio.mcp.push.interval:3000}")
-    private int mcpPushInterval;
+    
+    @Value("${nacos.istio.server.full:true}")
+    private boolean fullEnabled = true;
```

Comment:
What is the this param does for?

Reply:
determine whether to use Delta

Reply:
But this param never be used. consider to remove it.

Reply:
This parameter is used in incremental push. If the server does not agree to incremental push even if the client support, incremental push can not be carried out